cpp/src/ray/test/cluster/counter.h

```diff
@@ -0,0 +1,19 @@
+
+#pragma once
+
+#include <ray/api.h>
+
+using namespace ::ray::api;
```

Comment:
Is this `using namespace` needed?
Also it might be better to wrap this in a namespace. Same for `plus.h`

Reply:
If users want to use some APIs in this library, the `using namespace` is needed. But now, I can delete this.

Reply:
Ok. This is a test header so it does not matter that much. I think it is not recommended to have `using namespace` in a header especially not inside another namespace, unless the intention is to explicitly export those symbols with the header (e.g. https://google.github.io/styleguide/cppguide.html#Namespaces forbids `using namespace`).